controllers/grid/catalogEntry/IdentificationCodeGridHandler.php

```diff
@@ -364,18 +370,14 @@ public function deleteCode($args, $request)
 
         $identificationCodeDao = DAORegistry::getDAO('IdentificationCodeDAO'); /** @var IdentificationCodeDAO $identificationCodeDao */
         $identificationCode = $identificationCodeDao->getById($identificationCodeId, $this->getPublication()->getId());
-        if ($identificationCode != null) { // authorized
-
-            $result = $identificationCodeDao->deleteObject($identificationCode);
```

Comment:
Here's the problem, some methods are returning something at the `delete*()`, others not, and some partially :)